alerts/cluster-network-operator/NorthboundStaleAlert

```diff
@@ -0,0 +1,22 @@
+# Northbound Stale Alert
+
+## Meaning
+
+The alert `NorthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
+without succesfully synching any changes to the Northbound Database.
+
+## Impact 
+
+Networking control plane is degraded and Networking Configuration updates applied to the cluster will not be applied.
+Existing workloads should continue to have connectivity, but the OVN-Kubernetes control plane and/or the OVN southbound database
+may not be functional. 
+
+## Diagnosis
+
```

Comment:
We want to make sure that other alerts arent causing this alert to fire so can you say here first fix alerts `NoRunningOvnMaster` and `NoOvnMasterLeader` if they are firing before proceeding.